src/components/organisms/AdminVenueView/components/EventsView/CsvImportModal.tsx

```diff
@@ -0,0 +1,219 @@
+import React, { useCallback, useState } from "react";
+import { Modal } from "react-bootstrap";
+import { useAsyncFn } from "react-use";
+import classNames from "classnames";
+import { ParseError, ParseMeta } from "papaparse";
+
+import { importEventsBatch, ImportEventsBatchResult } from "api/events";
+
+import { parseCsv, ParseData, transformImportToEvents } from "utils/csv";
+
+import { ButtonNG } from "components/atoms/ButtonNG/ButtonNG";
+import {
+  FileButton,
+  FileButtonOnChangeData,
+} from "components/atoms/FileButton/FileButton";
+
+import "./CsvImportModal.scss";
+
+export interface CsvImportModalProps {
+  onHide: () => void;
+  show: boolean;
+}
+
+export const CsvImportModal: React.FC<CsvImportModalProps> = ({
+  onHide,
+  show,
+}) => {
+  const [parsedData, setParsedData] = useState<ParseData[] | undefined>();
+  const [parsedMeta, setParsedMeta] = useState<ParseMeta | undefined>();
+  const [parsedErrors, setParsedErrors] = useState<ParseError[] | undefined>();
+  const [isParsing, setIsParsing] = useState<boolean>(false);
+  const [importResult, setImportResult] = useState<ImportEventsBatchResult>();
```

Comment:
Why are the other states with `| undefined` but this one is not? How is this any different than the rest? Just wondering.